src/docs/sdk/performance/frames-delay.mdx

```diff
@@ -0,0 +1,145 @@
+
+On Mobile, slow and frozen frames are well-established metrics for tracking the responsiveness of
+applications. A phone or tablet typically renders 60 frames per second (fps). At 60 fps, every frame
+has 16 or 16.67 ms to render: 
+
+* __Slow Frames__: Using 60 fps, slow frames are frames that take more than 16 ms (Android) or 16.67 ms
+(iOS) to render.
+* __Frozen Frames__: Frozen frames are frames that take longer than 700 ms to render.
+
+The major downside is that they lack essential information on how long they lasted, which is a
+significant problem when prioritizing which unresponsive application parts need improvement. The following
+scenarios highlight the problem when prioritizing by looking at slow and frozen frames.
+
+Scenario | Slow Frames | Frozen Frames |
+-- | -- | -- | --
+1 |  5 all 500ms | 0 | 
+2 | 10 all 20ms | 0 | 
+3 |  0 | 1 with 800ms |
+
+When comparing the different scenarios, a user might want to fix scenario 3 first because it contains one frozen
+frame. When comparing scenarios 1 and 2, they would pick scenario 2 cause it has more slow frames. While scenario
+2 looks worse, the user experience is way worse for scenario 1 because the slow frames last longer.
+
+That's where a new metric called __frames delay__ comes to the rescue. Frames delay represents the delayed frame
+render duration of all frames. The frame delay for one frame is `actual frame duration - expected frame duration`.
+For example, with 60 fps the expected frame duration is 16.67 ms. When a frame takes 20 ms to render,  the frame
+delay is `20ms - 16.67ms = 3.33ms`. 
+
+Let's calculate the frames delay for scenarios 1 - 3.
+
+Scenario | Calculation | Frames Delay
+-- | -- | --
+1 | (500ms - 16.67ms) * 5 | 2416.65ms
+2 | (20ms - 16.67 ms) * 10 | 33.3ms
+3 | 800ms - 16.67ms | 783.33ms
+
+Now, when prioritizing the user-perceived degraded responsiveness by frames delay, we get  1, 3, 2 instead of
+3, 2, 1 when looking at slow and frozen frames.
+
+## Specification
+
+Frames delay represents the delayed frame render duration of all frames. The frame delay for one frame is
+`actual frame duration - expected frame duration`. For example, with 60 fps the expected frame duration is 16.67 ms.
+When a frame takes 20 ms to render,  the frame delay is `20ms - 16.67ms = 3.33ms`. As the expected frame rate can
+change at any time, SDKs must check the expected frame rate for every frame. The frame delay for a span is the sum
+of frame delays for all slow and frozen frames during the start and end time of the span.
+
+SDKs should attach frames delay in seconds to spans via <Link to="/sdk/performance/span-data-conventions/#mobile">span data</Link>.
+For transactions SDKs should send frames delay via the root span with span data.
+While it makes sense to calculate frames delay in milliseconds, __SDKs must send frames delay in seconds__, as SDKs already
+use seconds for most timestamps. For the specification here, we use milliseconds cause they are easier to read.
```

Comment:
m: Although it says "frame delays should be added to spans and transactions", I'm missing an explanation about when to start and end the measurement of the frame delay. We take it for granted that it is the duration of the span/transaction itself, but it's not so obvious.
Later in the document, in the scenarios, we have these graphs where it says `[---time interval---]`, but nowhere in the document does it specify what this "time interval" refers to.

Reply:
Good catch, thanks 👏